popUp.js

```diff
@@ -0,0 +1,148 @@
+const projects = [
+  {
+    title: 'Tonic',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
+    img: 'image/Snapshoot-Portfolio.jpg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    btn1: 'See live',
+    btn2: 'See Source',
+  },
+  {
+    title: 'Multi-Post Stories',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
+    img: 'image/Snapshoot-Portfolio-2.jpg"',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    btn1: 'See live',
+
+    btn2: 'See Source',
+  },
+  {
+    title: 'Tonic',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
+    img: 'image/Snapshoot-Portfolio.jpg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    btn1: 'See live',
+    btn2: 'See Source',
+
+  },
+  {
+    title: 'Multi-Post Stories',
+    img: 'image/Snapshoot-Portfolio-2.jpg',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    btn1: 'See live',
+    btn2: 'See Source',
+
+  }];
+
+const section = document.querySelector('.cards');
+const pop = document.querySelector('#pop-window');
+
+projects.forEach((project, id) => {
+  const cards = document.createElement('div');
+  cards.classList.add('job');
+  cards.innerHTML = (`
+        <img class="projetc-image" src= "${project.img}">
```

Comment:
- [ ] Please debug the image rendering code because it's not showing the image. Please look at the screenshot below

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/233674743-1acd9bc5-a88b-45ab-a63a-d3955fd0fc35.png)

Reply:
@MahmoudBakr23 Hello, I checked the Project it is showing the image on my side, even when I clone it is showing, kindly check again or clarify for me.

Reply:
@MahmoudBakr23 Hello, is the imaging showing now?